lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
Have you tried setting `env.ATOM_SHELL_INTERNAL_RUN_AS_NODE = 1`?

That should allow `process.execPath` to work from Atom I would think.

https://github.com/atom/atom/blob/1955a549a2150de9f881e9ecdd6b68370dc6531a/src/buffered-node-process.coffee#L49